packages/playground/_stories/main/Dialog/Dialog.stories.ts

```diff
@@ -120,6 +120,29 @@ FioriDialog.args = {
 </div>`,
 };
 
+export const BarInDialog = Template.bind({});
+BarInDialog.storyName = "Bar in Header/Footer";
+BarInDialog.args = {
+	header: `<ui5-bar slot="header" design="Header">
+				<ui5-title level="H5" slot="startContent">Bar used in Header and Footer</ui5-title>
+				<ui5-button class="dialogCloser" design="Transparent" id="closeDialogButton" slot="endContent" icon="decline"></ui5-button>
+			</ui5-bar>`,
```

Comment:
this indentation should be corrected to so it looks right in the Storybook page:
![image](https://user-images.githubusercontent.com/22086365/230628919-c72f60a1-9960-4c14-96a8-0166c7ca23bb.png)


We can't make the indentation look ok both in Storybook and in TS, so we choose to make it look correct only for Storybook